addon/validators/format.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * For code taken from ember-cp-validations
+ * Copyright 2016, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+
+import Ember from 'ember';
+import buildMessage from 'ember-changeset-validations/utils/validation-errors';
+
+const {
+  isEmpty
+} = Ember;
+
+const regularExpressions = {
+  "email": /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/,
```

Comment:
no need to have quotes around the keys, unless you have some strangely named key like `foo-bar`